game.py

```diff
@@ -20,7 +20,38 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    
+    snowman_word_dic = build_letter_status_dict(snowman_word)
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
```

Comment:
These two lines seem to do the exact same thing, and as I was going through the rest of the code, it will cause issues given how it is executed. I understand why you might be tempted to have two snowman dictionaries to hold separate information, but in this particular case, one is more than enough!